content/routing.md

```diff
@@ -32,7 +32,7 @@ meteor add kadira:flow-router
 
 *Snippet: Adding the Flow Router package*
 
-At the time of writing this guide, Flow Router is at version 2.x.
+At the time of writing this guide, [Flow Router](https://github.com/kadirahq/flow-router) is at version 2.x.
```

Comment:
I feel like that issue thread is probably the right place for people to have that discussion, no? I don't think it's possible to defend each design decision up front. Alternatively, if you put together an article about Flow Router's design decisions, I can link to it here.